src/ffi/cpython/pystate.rs

```diff
@@ -12,13 +22,38 @@ pub const PyTrace_C_EXCEPTION: c_int = 5;
 pub const PyTrace_C_RETURN: c_int = 6;
 pub const PyTrace_OPCODE: c_int = 7;
 
+// skipped PyTraceInfo
+// skipped CFrame
+// skipped _PyErr_StackItem
+// skipped _PyStackChunk
+// skipped _ts (aka PyThreadState)
+
 extern "C" {
```

Comment:
The cfgs in this entire block don't do anything because the entire module import is already gated with `#[cfg(all(Py_3_8, not(PyPy)))]` (https://github.com/PyO3/pyo3/blob/main/src/ffi/cpython/mod.rs#L43).

Reply:
As above will remove that gating and make the cfgs here meaningful.